### ISSUE TITLE
Be ale to modify openshift.io from the crd name

### ISSUE THREAD

**Issue by manuelbuil** [k8snetworkplumbingwg/sriov-network-operator#78]
The current name of the crds is not configurable and they are:
```
sriovibnetworks.sriovnetwork.openshift.io
sriovnetworknodepolicies.sriovnetwork.openshift.io
sriovnetworknodestates.sriovnetwork.openshift.io
sriovnetworks.sriovnetwork.openshift.io
sriovoperatorconfigs.sriovnetwork.openshift.io
```
it would be nice to be able to replace the string "openshift.io" by something more vendor neutral. In the same way as we can modify the resource prefix by using the env `RESOURCE_PREFIX`

**Comment by zshi-redhat** [comment]
@manuelbuil I think it's nice to have a neutral crd name like kubernetes.io, would you be able to work on a fix?

**Comment by manuelbuil** [comment]
> @manuelbuil I think it's nice to have a neutral crd name like kubernetes.io, would you be able to work on a fix?

I don't have bandwidth right now, but perhaps next week I could try ;)

**Comment by mrostecki** [comment]
I noticed that the `k8s.cni.cncf.io` CRD domain is already used in the other network plumbing wg projects like https://github.com/k8snetworkplumbingwg/network-attachment-definition-client

Maybe it would be more suitable than `kubernetes.io`?

**Comment by zshi-redhat** [comment]
> I noticed that the `k8s.cni.cncf.io` CRD domain is already used in the other network plumbing wg projects like https://github.com/k8snetworkplumbingwg/network-attachment-definition-client
> 
> Maybe it would be more suitable than `kubernetes.io`?

Agree, I later found out that `kubernetes.io` is reserved for k8s related crds. We might not be able to use it directly, so `k8s.cni.cncf.io` should not have the issue and it is consistent with other npwg projects.
